the-collab-lab/tcl-15-smart-shopping-list
# Issue #2 | Add react-router-dom and links

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓  | :sparkles: New feature     |
|  ✓ | :link: Update dependencies |

## Acceptance Criteria





## Description

* `react-router-dom` was added as a project dependency in order to create 2 links: "List" and "Add item"
*  3 new components were created to satisfy this functionality: `List`, `AddItem` and `Navigation` 
*  **Outcome**: the user is now able to navigate between the “list” view and the “add an item” view. The interface is clearly delineated between viewing the list content and adding a new item.

## Related Issue

Closes #2 

## Testing Steps / QA Criteria



* From your terminal, pull down this branch with git pull origin `ky-dg-react-router-dom` and check that branch out with git checkout `ky-dg-react-router-dom`
* Then `yarn` to install the newly added dependencies locally and `yarn start` to launch the app.
* Click on "List" (left) and "Add Item" (right) links at the very bottom of the page. Make sure that when the links are clicked, the browser URL updates to represent each link's component. Also, the selected link should be displayed in **bold**.
